### PR TITLE
378 - Added default blocks per sites

### PR DESCRIPTION
**Pull request by cmarianhodis**
Some issues with git and I had to delete my branch, hence the new pull request.

All requests from https://github.com/Lp-digital/FavoriteBundle/pull/1 were made. Including the suggestion of having only one entity from https://github.com/Lp-digital/FavoriteBundle/pull/1#discussion_r49600785

Of course you would still need to run the following command in order to update the database:
```bash
./backbee bundle:update fav --force
```

### CODE REVIEW

File: Entity/BookMark.php [hunk 1]
```diff
@@ -25,14 +25,17 @@
 
 /**
  * @author Mickaël Andrieu <mickael.andrieu@lp-digital.fr>
+ * @author Marian Hodis <marian.hodis@lp-digital.fr>
  *
- * @ORM\Entity()
+ * @ORM\Entity(repositoryClass="LpDigital\Bundle\FavoriteBundle\Repository\BookMarkRepository")
  * @ORM\Table(name="user_bookmarks")
  * @ORM\HasLifecycleCallbacks
```
Comment (mickaelandrieu) [inline Entity/BookMark.php]: was an old issue (from my side) but you still don't need this annotation: sorry :)

Comment (cmarianhodis) [inline Entity/BookMark.php]: HasLifecycleCallbacks you mean ?

Comment (mickaelandrieu) [inline Entity/BookMark.php]: yes :)

**Comment by mickaelandrieu** [comment]
By the way, nice job @cmarianhodis ! My vote doesn't count anymore but :+1:  :p

File: Entity/BookMark.php (hunk 1 above)
Comment (cmarianhodis) [inline Entity/BookMark.php]: Somehow that passes by me everytime, I removed it now

**Comment by cmarianhodis** [comment]
It doesn't count officially let's say, but still counts :)

**Comment by crouillon** [comment]
yeah, still counts @mickaelandrieu ;-)
Thanks to both of you
:+1:

**Pull request merged**